langchain4j-vearch/src/test/java/dev/langchain4j/store/embedding/vearch/VearchEmbeddingStoreIT.java

```diff
@@ -24,15 +16,9 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.data.Percentage.withPercentage;
 
-@Testcontainers
 class VearchEmbeddingStoreIT extends EmbeddingStoreIT {
 
-    @Container
-    static GenericContainer<?> vearch = new GenericContainer<>(DockerImageName.parse("vearch/vearch:3.4.1"))
-            .withExposedPorts(9001, 8817)
-            .withCommand("all")
-            .withCopyFileToContainer(MountableFile.forClasspathResource("config.toml"), "/vearch/config.toml")
-            .waitingFor(Wait.forLogMessage(".*INFO : server pid:1.*\\n", 1));
+    static VearchContainer vearch = new VearchContainer();
```

Comment:
Are we also migrating from testcontainers in this MR? thanks!

Reply:
I don't understand, it's still using `Testcontainers` to implement. I encapsulate the `VeachContainer` to reuse just in case.

Reply:
ah, i missed that it is your new class, looks fine then, thanks for clarification!